### ISSUE TITLE
[scroll-animations-1] Do we need container-name references?

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#7046]
Someone raised the question of whether we need the container-name reference in the scroll() notation in the #6674 proposal, or if the ability to name scroll timelines using the scroll-timeline properties is enough.

Note: @mirisuzanne and I had included both in our proposal because it wasn't clear which one was better, or if we in fact wanted both.

**Comment by mirisuzanne** [comment]
I feel the other way about css-naming vs the `selector()` function. I like the fact that I can create names _in CSS_ that I want to be able to reference in CSS, without needing those names to exactly match something in the DOM. I don't like digging through HTML to find and apply new selectors in various places, just for the sake of a presentational naming that is really specific to the current styles. 

I do get the point about a proliferation of `*-name` properties, and in some ways those could all be merged into a single CSS naming property. The advantage of having distinct ones is that they can cascade in parallel, instead of overriding each other? That's been a big discussion in relation to Container Queries recently. But it sure would feel _neater_ if we didn't need a whole list of similar properties.

**Comment by bramus** [comment]
Could be a spin-off feature, to think about a generic `-name` property that could be used by many properties that target a `<custom-ident>`.

As far as I'm concerned, this issue may be closed.
